src/Core/src/Platform/Android/Resources/values/strings.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<resources>
-    <string name="app_name">Xamarin</string>
-    <string name="action_settings">Settings</string>
```

Comment:
I did not find anything that actually used these. Delete them?

Reply:
It looks like that was part of the .NET MAUI samples we were originally using

https://github.com/dotnet/maui/commit/3aa86b4c988355203998fb9550dd85d7c266ad38#diff-7c01aad57ec768e8b14c209f0ddfbe8ed735c8741698e3b11bac8cf71d65d145R3

I don't see a reference to this at all over on the XF repository so I'd say fine to delete